### PR TITLE
490:Fix for driver UI on NodePort to be accessible when SSL enabled

### PR DESCRIPTION
**Pull request by breetasinha1109**
Fix to support SSL enabled driver UI.

### CODE REVIEW

**Comment by breetasinha1109** [comment]
@liyinan926 Hi, I have pushed the changes for the fix. What are the next steps to be done?

File: pkg/controller/sparkapplication/sparkui.go [hunk 1]
```diff
@@ -145,6 +140,16 @@ func createSparkUIService(
 // in Spec.SparkConf if it is present, otherwise the default port is returned.
 // Note that we don't attempt to get the port from Spec.SparkConfigMap.
 func getUITargetPort(app *v1beta1.SparkApplication) string {
+        var (
+        sparkUIPortConfigurationKey = "spark.ui.port"
+        defaultSparkWebUIPort = "4040"
+        )
+
+        if app.Spec.SparkConf["spark.ssl.ui.enabled"] == "true" {
```
Comment (liyinan926) [inline pkg/controller/sparkapplication/sparkui.go]: Please run `go fmt` to format the code correctly.

**Comment by liyinan926** [comment]
@breetasinha1109 are you still interested in getting this PR merged?

**Comment by breetasinha1109** [comment]
@liyinan926 Testing is still in progress. Once everything is completed, I will update on this.
Thanks.

**Pull request closed without merging**